Source/WebKit/PlatformGTK.cmake

```diff
@@ -10,6 +13,10 @@ if (USE_SKIA)
     include(Platform/Skia.cmake)
 endif ()
 
+if (NOT DEFINED ENV{SWIFTC})
+    message(FATAL_ERROR "Must define SWIFTC environment variable to point to swiftc")
+endif ()
```

Comment:
Is this check needed?

My understanding is that when you declare Swift as a language in the `project(WebKit)` line, CMake will try to automatically find the Swift compiler (and use the `SWIFTC` environment variable as a hint) and fail the configuration step if it cannot.